pages/dkp/kommander/2.0/gpu/index.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: layout.pug
+navigationTitle: GPUs
+title: GPUs
+menuWeight: 150
+excerpt: Configure GPU for Kommander cluster
+beta: false
+enterprise: false
+---
+
+<!-- markdownlint-disable MD004 MD007 MD025 MD030 -->
+
+This section describes the Nvidia GPU support on Kommander. This document assumes familiarity with Kubernetes GPU support. AMD GPUs are currently not supported.
```

Comment:
Why do we specifically name AMD GPUs here? Would it be more general to say `Currently Kommander only supports Nvidia GPUs.`